driver/mobile/keyboard.go

```diff
@@ -0,0 +1,26 @@
+package mobile
+
+import (
+	"fyne.io/fyne"
+)
+
+// Keyboard represents a standard fyne keyboard
```

Comment:
This doc line could probably be more descriptive. I also wonder if the type name should be a little clearer?

Reply:
KeyboardLayout maybe?

Reply:
I think that would work. Though I do wonder if this is truly a layout - people might associate that with QWERTY/AZERTY ;)